src/components/Checkbox.vue

```diff
@@ -64,15 +64,16 @@ export default {
     }
   },
   created() {
-    this.localModel = this.value;
+    this.localModel = this.modelValue;
   },
   watch: {
-    value(newValue) {
+    modelValue(newValue) {
       this.localModel = newValue;
     }
   },
   methods: {
-    handleChange() {
+    handleChange(event) {
+      this.$emit('update:modelValue', event.target.checked);
       this.$emit('input', this.localModel);
```

Comment:
Why are we still emitting the input event if it's no longer used?

Reply:
I left the old input emmission in in case a project uses it (e.g some parent is listening for it) to keep the API as similar as possible. Not sure that anything is though